src/base.css

```
@@ -598,6 +598,14 @@
 		background: var(--ins-bg);
 		text-decoration: underline;
 	}
+	ins.diff-inactive,
+	del.diff-inactive {
+		all: unset;
+	}
+	ins:not(.diff-inactive) a,
+	del:not(.diff-inactive) a {
```

Alternatively, maybe this should be:
```suggestion
	ins:not(.diff-inactive) *,
	del:not(.diff-inactive) * {
```
Links aren't the only things that change the color of the text, and all things things are hard to notice as being part of an ins or del if they're not green or red. As this goes away and the normal colors come back in the "current" and "future" views, I think the tradeoff is reasonable: in diff view, you most easily seen what has been added/removed, and in other views, you get the normal styling for better legibility.